ui/appui-react/src/appui-react/widgets/WidgetDef.tsx

```diff
@@ -537,16 +541,97 @@ export class WidgetDef {
    * I.e. opens the stage panel or brings the floating widget to front of the screen.
    * @public
    */
-  public show() {
+  public async show() {
```

Comment:
Having this change to a Promise<void> will cause users to have linter warnings that they should listen to the result, which they will not ever really care about. You should be able to refactor this to get the storage setting and then using the result directly with a `.then` or even `.finally` method, which would remove the requirement of the `async`.

Reply:
Removed async and utilized .then